3.3/.sti/bin/run

```diff
@@ -2,11 +2,11 @@
 source /opt/openshift/etc/generate_container_user
 
 function is_gunicorn_installed() {
-  pip show -q gunicorn
+  hash gunicorn &>/dev/null
```

Comment:
http://stackoverflow.com/questions/592620/check-if-a-program-exists-from-a-bash-script/677212#677212

`hash` is a bash builtin and has its advantages over `which`. Agree with you, `which` was the first thing in my mind too.